scala-interpreter/build.sbt

```diff
@@ -18,3 +18,4 @@ import sbt.Tests.{Group, SubProcess}
  */
 
 libraryDependencies ++= Dependencies.sparkAll.value
+libraryDependencies += "com.github.jupyter" % "jvm-repr" % "0.1.0"
```

Comment:
What does this do? Looking it up, I see that @rdblue was the author, but now it's under the jupyter umbrella. BSD-3 clause should be fine.

Is there a reason it's just used here and not all interpreters?